src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let amount = 0;
+
+  if (days < 3) {
+    amount = days * 40;
+
+    return amount;
+  }
+
+  if (days < 7) {
+    amount = (days * 40) - 20;
+
+    return amount;
+  }
+
+  return (days * 40) - 50;
```

Comment:
what a magic numbers 🪄
lets create variables and then use them
it is better for managing them from one place and for understanding what do they mean